connector/countconnector/README.md

```diff
@@ -0,0 +1,213 @@
+# Count Connector
```

Comment:
I like how this readme is written, mostly demonstrating how connectors work. At the same time, I wonder if this wouldn't be confusing to users of this connector specifically.

Reply:
Thanks, I'm reworking this into a general connectors readme and a dedicated count connector readme.